examples/directory-bucket/main.tf

```diff
@@ -0,0 +1,111 @@
+locals {
+  region  = "eu-west-1"
+  zone_id = "euw1-az1"
```

Comment:
Could you make AZ ID discoverable using a data source? There are cases where some AZs are not available in some accounts.